octobot_trading/exchanges/margin/margin_exchange.py

```diff
@@ -53,3 +53,6 @@ def get_name(cls) -> str:
 
     async def get_position_status(self, position_id, symbol=None):
         raise NotImplementedError("get_position_status is not implemented")
+
+    def cleanup_positions_dict(self, positions):
+        return positions
```

Comment:
WIP ?

Reply:
No it's the default behaviour. For example it's overwritten in bitmex MarginExchange tentacle.